scala/ppml/src/main/scala/com/intel/analytics/bigdl/ppml/attestation/AttestationCLI.scala

```diff
@@ -98,7 +98,7 @@ object AttestationCLI {
                     params.asURL.split(":")(1), params.appID, params.apiKey)
             case ATTESTATION_CONVENTION.MODE_BIGDL =>
                 new BigDLAttestationService(params.asURL.split(":")(0),
-                    params.asURL.split(":")(1), params.httpsEnabled)
+                    params.asURL.split(":")(1), params.appID, params.apiKey, params.httpsEnabled)
```

Comment:
why params.httpsEnabled is needed here, can we force httpsenabled to always be true